frontend/src/utils/og.ts

```diff
@@ -16,7 +16,13 @@ export const createDetailPageMeta = ({
   isSensitive: boolean
 }) => {
   const head: { meta: Meta } = { meta: [] }
-  const meta: Meta = []
+  const meta: Meta = [
+    {
+      key: "robots",
+      name: "robots",
+      content: "all",
+    },
+  ]
```

Comment:
Is this necessary? I thought by default the seo robots plugin was adding the correct robots meta tags to pages and that we could avoid setting these values outside of the nuxt config. That was the paradigm I tried to embrace in #4659, at least.

Reply:
I noticed that I incorrectly reversed the `seo` Playwright test changes from the single result fix PR. So, when fixing this, I re-added this value here.
Then I tested the set up again, and realized that the robots setup used a trailing slash for `disallow` routes and therefore was creating incorrect meta tags and robots settings. Updated in the latest commit. I'm not sure what the difference between the value `all` and the default value used by robots module is.